_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,455 @@
+---
+id: 9bd38c
+name: Content has non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when content is identified through a visual reference, there are also non-visual references identifying the same content.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Daniël Strik
+    - Jean-Yves Moyen
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [text node][] that is either [visible][] or [included in the accessibility tree][].
+
+## Expectation
+
+For each test target that includes at least one of the [visual reference words][], one of the following is true:
+
+- (**no indication**) the test target does not identify any [web content][] through the use of any [visual reference words][]; or
```

Comment:
Yes, that make sense. I tend to dislike the "except when" formulation, especially in long lists, since it's a negation and it's not always obvious how to parse it when mixed with conjunction/disjunction…

I'm also realising that the expectation does not really handle test targets that do not include any reference word…

How about:
> For each test target, either it contains no ref word, or it does not identify any content, or it does identify content through ref words and one of the following is true:

?